### ISSUE TITLE
Export Deck with Cursed Blade (Russian) - failed

### ISSUE THREAD

**Issue by Neohunterv** [Epix37/Hearthstone-Deck-Tracker#2156]
Hello
I try to export this deck http://www.hearthpwn.com/decks/392484-face-warrior-v-2-season-21-top-200-legend
BTW i manual create it in HDT because it not imported from WEB (but as i know site www.hearthpwn.com supported  )
When i try export this deck in HS - i see only ????????? in search  and not any card from deck not exported
And i see message in HDT - Export incomplete The following cards  were not found
and list all cards from deck
But all of them  i have. 

I think what problem with card  - Cursed Blade  (Проклятый клинок) in Rus
Pls fix this issue
10x

**Comment by Epix37** [comment]
Try enabling `options > tracker > exporting > paste names from clipboard`.

**Comment by Neohunterv** [comment]
Hm 
10x via clipboard work  - import from url and next export 
But why the same not work 
1. Import  - via from web ?
2. Export  - if deck create manually ?

**Comment by Epix37** [comment]
I'm not entirely sure what you mean, could you rephrase the problem? 
Step by step, what do you do, what do you expect to happen, what does actually happen?

**Comment by Neohunterv** [comment]
OK
Step by step 
Issue 1 - import from http://www.hearthpwn.com/decks/392484-face-warrior-v-2-season-21-top-200-legend
1. I create new warrior constructed deck  
2. Than Import, From Web - inserted full url  - http://www.hearthpwn.com/decks/392484-face-warrior-v-2-season-21-top-200-legend
3. Press ok
4. Get message - Error Could not load deck from specified url 
But the same deck via your instruction successfully imported, via import From Web  - fail, why ?
I think this error.

Issue 2 - Fail export  deck http://www.hearthpwn.com/decks/392484-face-warrior-v-2-season-21-top-200-legend - if i create it manually in HDT
1. I create new warrior constructed deck  
.....
I found exactly what problem
If i save deck in HDT and in name deck present Russian characters  - when i export this deck in HS - error 
Also even if in name deck not present Russian character (only English character)  but in the moment of export deck  - current active keyboard layout Russian - also problem with import
In this case we see in HS in field name deck - 'v', and in search field in HS also in each card try entered  - 'v' and nothing founded
I think problem not only with Russian characters and active keyboard layout i think its with all not English characters and not English current active keyboard layout.

**Comment by Epix37** [comment]
Issue 1:

Seems to be working for me, possibly also a language problem. 
Could you reproduce the problem one more time and then post the hdt log? [See here](https://github.com/Epix37/Hearthstone-Deck-Tracker/wiki/Reporting-Issues).

Sidenote: You can `import - from web` right away, without manually creating the deck first.

**Comment by Neohunterv** [comment]
OK
Issue 1
Reproduce problem again  - see in log 11:35 
http://pastebin.com/SwNAQbDZ

BTW Question - how i can remove HDT data folder to other disk (now located in system SSD disk )

**Comment by Epix37** [comment]
I think you are having the same underlying issue as #2115.

You can move the data folder by:
1. unchecking `options > tracker > settings > save data in appdata`
2. setting the new path via `options > tracker > settings > set data path`

**Comment by Neohunterv** [comment]
OK thanks for answer  - i resolve issue 1 - its local firewall problem
So what about issue 2 - with non English characters ?

**Comment by Epix37** [comment]
Second issue will be fixed with the next update.  (#2260)
